src/AutoComplete/AutoComplete.js

```diff
@@ -266,6 +266,9 @@ class AutoComplete extends Component {
   };
 
   handleItemTouchTap = (event, child) => {
+    // Prevent ghost clicks
+    event.preventDefault();
```

Comment:
I'm working on a similar pull request for https://github.com/callemall/material-ui/issues/6938 and I'm wondering if you could be a little clearer what you mean here? Would you mean doing something like passing the `event` object to the exposed `onNewRequest` property?